.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
     - ATHENS_MONGO_CONNECTION_STRING=mongodb://127.0.0.1:27017
     - CODE_COV=1
     - ATHENS_DIR=${GOPATH}/src/github.com/gomods/athens
+    - REGISTRY=carolynvs/ # Temporary workaround until we have access to the gomods/ registry
```

Comment:
As soon as we have travis's environment variables updated with our godmods account creds, we can switch this back to `gomods/`.